app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def calculate_team_total_rating(team: list) -> int:
+    return sum(player.get_rating() for player in team)
+
+
+def elves_concert(elves: list) -> None:
+
+    for elf in elves:
+        elf.play_elf_song()
+
+
+def feast_of_the_dwarves(dwarves: list) -> None:
```

Comment:
You should specify inner type of list
```suggestion
def feast_of_the_dwarves(dwarves: list[Dwarf]) -> None:
```

Reply:
do it